pkg/reconciler/v1alpha1/taskrun/validate_test.go

```diff
@@ -0,0 +1,267 @@
+package taskrun_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/knative/build-pipeline/pkg/apis/pipeline/v1alpha1"
+	"github.com/knative/build-pipeline/test"
+	duckv1alpha1 "github.com/knative/pkg/apis/duck/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
```

Comment:
for completeness, can we add a test for valid TaskRun and then make sure validateTaskRun does not return err.
Ignore if this comment if this is already tested somewhere else.